### PR TITLE
feat: Added a 'More from publisher' section to the Marketplace's 'App Details' view

### PR DESCRIPTION
**Pull request by pedroCloudoki**

### CODE REVIEW

**Review by s1moe2 — CHANGES_REQUESTED** [review]

File: lib/pages/AppDetails/AppDetails.tsx [hunk 1]
```diff
@@ -138,6 +142,57 @@ const AppDetails: React.FC<AppDetailsProps> = ({
...
+      that can be used by our App Catalog component. */
+      const appsFromPublisher = filteredMarketplaceApps
+        .filter((app) => {
+          return app.id !== selectedAppDetails.id
+        })
+        .slice(0, 3)
+        .map((app) => {
+          return {
+            appDescription:
+              app.shortDescription.length > 0
```
Comment (s1moe2) [inline lib/pages/AppDetails/AppDetails.tsx:177]: This condition is used in 2 other places. Needs to be extracted and refactored (and possibly changed to non-ternary syntax).

**Review by s1moe2 — DISMISSED** [review]

File: lib/pages/Marketplace/Marketplace.tsx [hunk 2]
```diff
@@ -76,20 +84,7 @@ const Marketplace: React.FC<MarketplaceProps> = ({
...
-              ? app.description
-              : t('appMarketplace.noDescriptionAvailableText'),
-          appID: app.id,
-          appLabels: app.labels,
-          appLogo: app.logo,
-          appName: app.name,
-          appPublisher: app.organization.name,
-          appUpdatedAt: app.updatedAt,
-        }
+        return appDetailsMapping(app)
```
Comment (a31859) [inline lib/pages/Marketplace/Marketplace.tsx:87]: Move to saga like it was done for the publisher.

File: lib/pages/Marketplace/Marketplace.tsx [hunk 3]
```diff
@@ -243,20 +238,7 @@ const Marketplace: React.FC<MarketplaceProps> = ({
...
-              ? filteredApp.description
-              : t('appMarketplace.noDescriptionAvailableText'),
-          appID: filteredApp.id,
-          appLabels: filteredApp.labels,
-          appLogo: filteredApp.logo,
-          appName: filteredApp.name,
-          appPublisher: filteredApp.organization.name,
-          appUpdatedAt: filteredApp.updatedAt,
-        }
+        return appDetailsMapping(filteredApp)
```
Comment (a31859) [inline lib/pages/Marketplace/Marketplace.tsx:241]: Move to saga like it was done for the publisher.

**Review by a31859 — CHANGES_REQUESTED** [review]

**Review by a31859 — APPROVED** [review]

**Pull request merged**